projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/integration-card/style.scss

```diff
@@ -78,14 +78,19 @@
 
 	&__plugin-badge,
 	&__connected-badge,
-	&__setup-badge {
+	&__setup-badge,
+	&__loading-badge {
 		order: 2;
 
 		@media (max-width: 782px) {
 			order: 3;
 		}
 	}
 
+	&__loading-badge {
+		border-radius: 4px;
```

Comment:
If we apply the style manually, let's apply it via the same variable the badge uses:

https://github.com/Automattic/wp-calypso/blob/trunk/packages/ui/src/badge/style.module.scss#L21

E.g. now it's 4px while badge is 2px.

Although I think we might be able to [just use the Badge and animate it.](https://github.com/Automattic/jetpack/pull/45565/files#r2447878463)